modules/bmo-about-the-bare-metal-operator.adoc

```diff
@@ -0,0 +1,69 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_module-type: CONCEPT
+[id="bmo-about-the-bare-metal-operator_{context}"]
+= About the Bare Metal Operator
+
+Use the Bare Metal Operator (BMO) to provision, manage, and inspect bare-metal hosts in your cluster. 
+ 
+The BMO uses three resources to complete these tasks: 
+
+* `BareMetalHost`
+* `HostFirmwareSettings`
+* `FirmwareSchema`
+
+The BMO maintains an inventory of the physical hosts in the cluster by mapping each bare-metal host to an instance of the `BareMetalHost` custom resource definition. Each `BareMetalHost` resource has hardware, software, and firmware details specific to a bare-metal host in your cluster. 
+
+The BMO interfaces with bare-metal hosts in the cluster by using Metal3's Ironic service. The Ironic service uses the Baseboard Management Controller (BMC) on the host to interface with the machine. The BMO continually inspects the bare-metal hosts in your cluster to ensure each `BareMetalHost` resource accurately reflects the hardware, software, and firmware details for each bare-metal host in the cluster. 
+
+The BMO also uses the `HostFirmwareSettings` resource and the `FirmwareSchema` resource to detail firmware specifications for the bare-metal host. 
+
+Some common tasks you can complete by using the BMO include the following:
+
+* Provision bare-metal hosts to the cluster with a specific image
+* Format a host's disk contents before or after provisioning.
+* Turn on or off a host
+* Change firmware settings
+
+DRAFT IMAGE: 
+
+image::BMO-Diagram-v2.png[DRAFT - PLACEHOLDER IMAGE]
+
+.BareMetalHost
+
+The `BareMetalHost` resource defines a physical host and its properties. When you provision a bare-metal host to the cluster, you must define a `BareMetalHost` resource for that host. For ongoing management of the host, you can inspect the information in the `BareMetalHost` or update this information. 
+
+The `BareMetalHost` resource features provisioning specifications such as the following:
+
+* URL of the operating system boot image
+* Provisioning state 
```

Comment:
Provisioning state is a part of the status, it's not something that users specify.

The expected power state is, maybe worth mentioning.

Reply:
I changed the lead-in sentences for these lists to "provisioning information" and "hardware information", instead of "provisioning specifications" and "hardware specifications". Perhaps this way we don't need to worry about if it's a status or a specification, it's all information that is relevant to provisioning/hardware at a high level. 

I did add "Desired power state" if that's a relevant one for provisioning